### ISSUE TITLE
Why track_running_stats=False? What's the advantage?

### ISSUE THREAD

**Issue by Joevaen** [MIC-DKFZ/nnUNet#370]
Hi, fabian!
During my work, I noticed `track_running_stats=False`? Could you tell me why you don't track the mean and variance in training? A lot of algorithm set this parameter as True. Could you please explain the reason for me? Thank you very much.
I don't know if open the parameter will improve its performance.
Best,
Joevaen.

**Comment by FabianIsensee** [comment]
You should probably mention that you are talking about the instance normalization layers. This parameter is off by default. It is on by default in batch normalization.
I doubt that most people set it to True for instance normalization, but you can always try it.
Best,
Fabian

**Comment by Joevaen** [comment]
Hi, fabian!
I find this parameter in source code and I noticed it is off here: `https://pytorch.org/docs/master/_modules/torch/nn/modules/batchnorm.html`.
![image](https://user-images.githubusercontent.com/46133615/97416190-1ae95500-1941-11eb-8438-37be3cf4af8e.png)

**Comment by FabianIsensee** [comment]
![image](https://user-images.githubusercontent.com/11302164/97418784-81eb1780-1909-11eb-8fb7-42f36d42846c.png)
![image](https://user-images.githubusercontent.com/11302164/97418839-992a0500-1909-11eb-94be-781546a6dc5c.png)

**Comment by FabianIsensee** [comment]
as I said

**Comment by Joevaen** [comment]
Sorry for a carelessness, I slide into the BN source code, thank you all the same!
Best,
Joevaen.
